flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -288,7 +295,7 @@ public Builder uidPrefix(String newPrefix) {
 
       // Distribute the records from input data stream based on the write.distribution-mode.
       DataStream<RowData> distributeStream = distributeDataStream(
-          rowDataInput, table.properties(), table.spec(), table.schema(), flinkRowType);
+          rowDataInput, this.props, table.spec(), table.schema(), flinkRowType);
```

Comment:
Nit: We don't use `this` to access instance variables typically other than in the constructors and places it's absolutely needed.

Reply:
updated.